### PR TITLE
Add $wgMaxCredits on ManageWiki per request T5011

### PR DESCRIPTION
**Pull request by Hispano76**
It needs thorough revision! It may have errors.

### CODE REVIEW

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -348,6 +348,17 @@
 		'section' => 'edit',
 		'help' => 'The number of days within which a person must make edits to be considered an "active" user.',
 	],
+	'wgMaxCredits' => [
+		'name' => 'Editing attribution',
+		'from' => 'mediawiki',
+		'restricted' => false,
+		'type' => 'integer',
+		'minint' => -1,
+		'maxint' => 0,
```
Comment (The-Voidwalker) [inline ManageWikiSettings.php]: This should probably be higher

Comment (RhinosF1) [reply ManageWikiSettings.php]: See https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:$wgMaxCredits

This allows for all or none but probably and also name probably should be ‘Number of Editors to Attribute in footer’?

**Comment by Hispano76** [comment]
> Can we add https://www.mediawiki.org/wiki/Manual:$wgShowCreditsIfMax at the same time?
check

**Comment by Hispano76** [comment]
> Can we add https://www.mediawiki.org/wiki/Manual:$wgShowCreditsIfMax at the same time?

check

**Comment by PF94** [comment]
i littearly made a phabricaptor request and now i'm semi-responsibale for the birth of a pull request :P

File: ManageWikiSettings.php (hunk 1 above)
Comment (PF94) [reply ManageWikiSettings.php]: my mistake, i think the limit should be set to say like, 25?

**Pull request merged**